new-dti-website/components/bottom.tsx

```diff
@@ -85,22 +85,22 @@ const Bottom: React.FC = () => (
           src="/images/outreach_1.svg"
           width={275}
           height={143}
-          alt="outreach 1"
+          alt="kids stacking boxes at make-a-thon"
```

Comment:
while we're at it adding alt text, do u think u could add alt text for the images on the Initiatives page? i think the images a missing a few or aren't too precise:


![Screenshot 2024-11-30 at 12 46 53 AM](https://github.com/user-attachments/assets/bd52a126-fe55-419a-87e3-e84d6454b502)
![Screenshot 2024-11-30 at 12 47 06 AM](https://github.com/user-attachments/assets/beaa0f5c-946c-46f7-8a90-48c4a57c7403)

![Screenshot 2024-11-30 at 12 47 23 AM](https://github.com/user-attachments/assets/eda2ec00-9a6d-4a78-a011-ae23a193ddf8)

Reply:
Some of my suggestions (feel free to edit as u see fit):

|Image|Alt text|
|-|-|
|![image](https://github.com/user-attachments/assets/e6f00c86-7aea-4d1e-b59c-36229c010dfd)|"a DTI member teaching members of the Ithaca community about technology"|
|![image](https://github.com/user-attachments/assets/f9d97a3c-cd02-4177-a5ce-b1e9ab62e5cd)|"a DTI member presenting a series of graphs at the Half-Baked event"|
|![image](https://github.com/user-attachments/assets/3aac1a23-ec98-4ff8-8558-76907d6435b4)|"a DTI member presenting code at the BigRed//Hacks hackathon event"|
|![image](https://github.com/user-attachments/assets/78a5d1b6-643a-4a9e-b9d8-4aad408ffa6b)|"DTI members posing in front of the Millennium logo"|